extensions/amp-brightcove/0.1/amp-brightcove.js

```diff
@@ -90,7 +90,7 @@ class AmpBrightcove extends AMP.BaseElement {
   }
 
   /** @override */
-  documentInactiveCallback() {
+  pauseCallback() {
```

Comment:
So, this class now needs `unlayoutOnPause` until we address that issue, right? Let's do this and add TODO and reference that bug#.